versioned_docs/version-3.0.0-beta/projects/domain-and-https/centralized-domain/attach-domain.md

```diff
@@ -0,0 +1,38 @@
+---
+id: attach-domain
+title: Attach Domain
+sidebar_label: Attach Domain
+slug: /domain-and-https/centralized-domain/attach-domain
+---
+
+## Steps to Attach a Domain
```

Comment:
same thing here, incorporate the changes asked in the quick start section for configuring domains